utils/README.md

```diff
@@ -0,0 +1,6 @@
+# Python utilities package
+
+`utils` package contains helper function used across the solution for local testing.
+
+> 📝 This package could be refactored, so that storage utilities are cleaned up and moved to
+> PySparkle.
```

Comment:
Does this comment need to be here, or perhaps we just need a ticket for doing this?

Reply:
A ticket is probably a better idea :)